tests/e2e-inst-signatures/scripts/ds_writer/ds_writer.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/aquasecurity/tracee/api/v1beta1"
+)
+
+func main() {
+	conn, err := grpc.Dial(
+		"unix:///tmp/tracee.sock",
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+	if err != nil {
+		panic(fmt.Errorf("failed to dial tracee grpc server: %v", err))
+	}
+	client := v1beta1.NewDataSourceServiceClient(conn)
+	_, err = client.Write(context.Background(), &v1beta1.WriteDataSourceRequest{
+		Id:        "demo",
+		Namespace: "e2e_inst",
+		Key:       structpb.NewStringValue("bruh"),
+		Value:     structpb.NewStringValue("moment"),
```

Comment:
Can we do just a bit more ? I mean, its a data source, maybe feed it with 100k entries and pick a single one ? Maybe a parallel write ? This is the **only** place we would be testing it in open source.

Reply:
done, tell me what you think